src/js/config/Config.template.js

```diff
@@ -14,6 +14,9 @@ module.exports = {
     },
     introduction: {
       enabled: false
+    },
+    'remove-oauth': {
+      enabled: false
```

Comment:
Isn't this where all plugins go? The installer has no internal plugins what-so-ever.